release-notes/swan-lake-release-notes/swan-lake-2201.3.0-release-note.md

```diff
@@ -231,7 +231,33 @@ Introduced the `--native` flag, which generates a GraalVM native executable when
 - The generated executable contains the modules in the current package, their dependencies, Ballerina runtime, and statically linked native code from the JDK.
 - `bal build --native`
 
+#### OpenAPI Tool
+- Introduced non-Ballerina IDL client declaration support for openAPI specifications. With the support, openAPI specifications can be imported(i.e. declared) at module level and, consumed as follows.
```

Comment:
```suggestion
- Introduced non-Ballerina IDL client declaration support to import OpenAPI specifications to declare and consume them at the module level as follows.
```

Reply:
@NipunaRanasinghe - Please check the changes.

Reply:
This seems incorrect as the client declaration is only getting declared (but not getting consumed) at the module level. Shall we modify accordingly?

Reply:
@lnash94 - Please change as follows:

- Introduced non-Ballerina IDL client declaration support for OpenAPI specifications. With this support, OpenAPI specifications can now be imported (i.e., declared) at the module level and consumed as follows.